app/i18n/fr/index.php

```diff
@@ -60,6 +60,7 @@
 		'bookmark_query' => 'Enregistrer la recherche courante',
 		'favorites' => 'Articles favoris (%s)',
 		'global_view' => 'Vue globale',
+		'grid_view' => 'Grid view',	// TODO
```

Comment:
```suggestion
		'grid_view' => 'Vue grille',
```

Reply:
Duplication of the same translations. Either for this PR or another, I would prefer to keep a single copy